Empire/Empire.cpp

```diff
@@ -855,6 +920,9 @@ float ProductionQueue::TotalPPsSpent() const {
     return retval;
 }
 
+/** TODO: Is there any reason to keep this method in addition to the more specific 
+ * information directly available from the empire?  This should probably at least be renamed
+ * to clarify it is non-stockpile output */
```

Comment:
To me, "Available" specifically means with stockpile considerations included. It's the output, or how much is produced each turn, plus anything additional accessible from a stockpile. Combined, they are the available PP.